homepage/templates/homepage/users/sign_in.html

```diff
@@ -0,0 +1,17 @@
+{% extends "homepage/app_layout.html" %}
+
+{% block content %}      
+
+{% load crispy_forms_tags %}
+
+<div class="container py-5">
+    <h1>Sign In</h1>
+    <form method="POST">

```

Comment:
Error message display is supposed to be here, no?

Reply:
Error messages are only partly implemented and will be fully implemented on a separate PR with other features
I've added a comment in the PR description for more clarity.